local-tests/tests/wrapped-keys/testFailEthereumSignTransactionWrappedKeyWithMissingParam.ts

```diff
@@ -66,11 +66,7 @@ export const testFailEthereumSignTransactionWrappedKeyWithMissingParam = async (
         id,
       });
     } catch (e: any) {
-      if (
-        e.message.includes(
-          'Error executing the Signing Lit Action: Error: Missing required field: toAddress'
-        )
-      ) {
+      if (e.message.includes('Missing required field: toAddress')) {
```

Comment:
Made this less specific since the exact string isn't important and this is now being properly prefixed consistently due to refactors in error handling in the LA in this PR